webui/apps/docs/src/content/docs/docs/howto/use_k8s.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy Eidolon AI with Kubernetes
+description: Learn how to deploy and manage Eidolon AI in a Kubernetes environment
+---
+
+Running Eidolon AI on Kubernetes unlocks powerful deployment and scaling capabilities. This guide walks you through setting up Eidolon AI in your Kubernetes cluster.
+
+## Prerequisites
+
+- A running Kubernetes cluster and `kubectl` CLI ([Get started with Minikube][minikube])
+- Basic familiarity with [Eidolon AI](https://www.eidolonai.com/docs/)
+
+## Quick Setup
+
+1. Clone the quickstart repo:
+   ```bash
+   git clone git@github.com:eidolon-ai/eidolon-quickstart.git
+   ```
+
+2. Create a namespace (optional for Make users):
+   ```bash
+   kubectl create namespace eidolon-quickstart
+   ```
+
+3. Set up API access tokens using Kubernetes secrets:
+   ```bash
+   # Option 1: From .env file
+   kubectl create secret generic eidolon --from-env-file=.env
```

Comment:
let's add `make .env` before this line

Reply:
it would also be good to add the `-n eidolon-quickstart namespace` by default and add a note that it is optional rather than the other way around. that way people blindly copy-pasting will still be successful.